docs/docs/overview/quickstart/get-started-with-python.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 10
+sidebar_label: Get started with Python
+---
+
+# Python Quickstart
+
+Welcome to the Python Quickstart guide for Novu, a powerful notification service that enables you to send multi-channel (SMS, Email, Chat, Push) notifications from your Python applications. In this Quickstart, you'll learn how to seamlessly integrate Novu into your Python project and perform various essential tasks. Let's get started!
+
+## Prerequisites
+
+Before diving into the Quickstart, make sure you have the following:
+
+- Python3 installed on your development machine.
+- A Novu account. If you don't have one, sign up for free at [web.novu.co](https://web.novu.co)
+
+You can also [view the completed code](https://web.novu.co) of this quick start in a GitHub repo.
+
+### Install and Set Up Novu in your Python Project
+
+First, you must install the Novu package in your Python project. You can install the Novu package in your project in two ways. Open your terminal and run the following command:
+
+Via pip
+
+```bash
+pip install novu
+```
+
+Via poetry
+
+```bash
+poetry add novu
+```
+
+Once installed, you can import Novu into your project and initialize it using your Novu account credentials. This step establishes a connection between your app and the Novu notification service.
+
+```python
+from novu.config import NovuConfig
+
+NovuConfig().configure("https://api.novu.co", "<YOUR_NOVU_API_KEY>")
+```
+
+Replace the `<YOUR_NOVU_API_KEY>` value with the authentic key from the **API Key** section of your [Novu Dashboard](https://web.novu.co/settings).
+
+<aside>
+🔑 Note: Please do not hardcode your credentials in a file in production. Use environment variables instead.
+
+</aside>
+
+## Set Up A Channel Provider
+
+A channel provider is a service that provides one or more notification functionality such as sending an email, SMS, push notification etc. Our [integration store](https://web.novu.co/integrations) includes four channels: Email, SMS, Chat, and Push. These channels have multiple providers associated with them.
+
+| Channel | Providers                                                           |
+| ------- | ------------------------------------------------------------------- |
+| Email   | MailGun, Mandrill, MailJet, Amazon SES, Sendgrid, Postmark, Netcore |
+| SMS     | Twilio, Amazon SNS, Plivo, SMS, SMSCentral, Kannel, Infobip, Termii |
+| Chat    | Mattermost, Slack, Microsoft Teams, Discord                         |
+| Push    | FCM, APNS, Expo                                                     |
+
+Only one provider can be **active** per **channel**. Connect any of your favorite providers to get started. The email channel comes with Novu's email provider, which is active by default and includes 300 credits.
```

Comment:
Only one provider can be **active** per **channel** at a time. Connect anyone of your preferred providers to get started. The email channel comes with Novu's email provider, which is active by default and includes 300 credits.